src/aws/bin/terraform

```diff
@@ -2,12 +2,31 @@
 
 set -eo pipefail
 
+# Get current ngrok url from ngrok api launched with command "make ngrok"
+declare NGROK_URL
+declare DIR_NAME
+
+DIR_NAME=$(dirname "$0")
+NGROK_URL=$("$DIR_NAME"/../../../bin/get_ngrok_url)
```

Comment:
If somebody has a nicer way to do this…

Reply:
maybe https://github.com/openfun/richie-site-factory/blob/master/bin/upgrade#L5 ?